# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../runit-shutdown/forced_reboot_halt.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- runit-shutdown-0.2/src/halt.sh.orig	2006-04-09 01:53:16.000000000 +0200
+++ runit-shutdown-0.2/src/halt.sh	2006-04-09 01:55:07.000000000 +0200
@@ -27,7 +27,11 @@
 done
 
 if [ "$force" ]; then
-	echo "forced $PROG not implemented, ignored."
+	if [ "$PROG" == 'reboot' ]; then
+		exec /command/forced_reboot
+	else
+		exec /command/forced_poweroff
+	fi
 elif [ "$wtmp" ]; then
 	echo "wtmp record-only for $PROG not implemented, skiped."
 	exit 0
